packages/cards-editor-middleware/src/utilities.ts

```diff
@@ -0,0 +1,191 @@
+import { statSync, readdirSync, readFileSync } from 'fs';
+import { ToolsLogger, LogLevel } from '@sap-ux/logger';
+import { join } from 'path';
+
+interface MultiCardsPayload {
+    type: string;
+    manifest: object;
+}
+
+interface MultiCardsStringManifest {
+    integration: string;
+    adaptive: string;
+}
+
+export interface I18nEntry {
+    key: string;
+    value: string;
+    comment?: string;
+}
+
+export const CARD_TYPES = {
+    INTERGATION: 'integration',
+    ADAPTIVE: 'adaptive'
+};
+
+/**
+ * Return the version from package.json.
+ *
+ *  @returns - version from package.json
+ */
+function getVersion(): string {
+    let version = '';
+    try {
+        version = JSON.parse(
+            readFileSync(join(__dirname, '../../package.json'), { encoding: 'utf8' }).toString()
+        ).version;
+    } catch (error: any) {
+        const logger = new ToolsLogger({
+            logLevel: LogLevel.Info
+        });
+        logger.warn(`Could not read version from 'package.json'`);
+        logger.debug(error);
+    }
+    return version;
+}
```

Comment:
Which `package.json` is read here? I assume the one from this module `@sap-ux/cards-editor-middleware`?
If yes, `../../package.json` points to the wrong location.

 To read the `package.json` we could also just import it, like:
```typescript
import packageJson from '../../package.json';

function getVersion(): string {
    return packageJson.version;
}
```

Reply:
done.